inc/saiswitch.h

```diff
@@ -3518,6 +3518,15 @@ typedef enum _sai_switch_asic_sdk_health_category_t
  */
 typedef enum _sai_switch_stat_t
 {
+    /** Switch stat range start */
+    SAI_SWITCH_STAT_START,
+
+    /** Global (switch-wise) counter of packets trimmed but dropped due to failed shared buffer admission on a new queue (for trimmed packets) */
+    SAI_SWITCH_STAT_TRIM_DROPPED_PACKETS = SAI_SWITCH_STAT_START,
```

Comment:
Please change it to
SAI_SWITCH_STAT_DROPPED_TRIM_PACKETS

Reply:
done